app/controllers/teams_controller.rb

```diff
@@ -1,15 +1,15 @@
 class TeamsController < ApplicationController
   before_action :set_team, only: [:show, :edit, :update, :destroy, :assign_ownership, :delete_membership, :delete_ownership, :perform_action_on_multiple_members, :assign_membership_by_email]
   before_action :authenticate_user!, only: [:new, :create, :edit, :update, :destroy]
-  authorize_resource :team, only: [:index, :show, :new, :edit, :create, :update, :destroy]
+  load_and_authorize_resource :team, only: [:index, :show, :new, :edit, :create, :update, :destroy]
 
   # GET /teams
   def index
     multi_teams = Team.multiplayer
     if params[:filter] == "true"
-      @teams = multi_teams.includes(:team_members).where(team_users: { user_id: current_user })
+      @teams = @teams - (@teams - multi_teams.includes(:team_members).where(team_users: { user_id: current_user }))
```

Comment:
@jonasnoki Could you please explain, why this expression has to be like that?

Reply:
@jonasnoki I guess you just want to make `@teams` the intersection of (the `load_and_authorize`d) `@teams` and `multi_teams.includes(…)`? You could just try using the `&` operand of Ruby‘s Arrays then which should do the same.

Reply:
@bakoe got it right. & operator works like a charm.